lib/utils/async.html

```diff
@@ -67,13 +64,15 @@
        * delay.
        *
        * @memberof Polymer.Async.timeOut
-       * @param {number} delay Time to wait before calling callbacks in ms
+       * @param {number=} delay Time to wait before calling callbacks in ms
        * @return {AsyncInterface} An async timeout interface
        */
       after(delay) {
-        return  {
+        return {
           run(fn) { return setTimeout(fn, delay); },
-          cancel: window.clearTimeout.bind(window)
+          cancel(handle) {
```

Comment:
This recently changed on master to add `@function` to the doc. Have to recheck if the analyzer can still understand it.